knative-kubecon/README.md

```diff
@@ -268,7 +268,8 @@ servicemesh, which is listening on `$(minishift ip):32380`). Stringed together w
 command:
 
 ```bash
-$ curl -H "Host: dumpy.myproject.example.com" "http://$(minishift ip):32380/health"
+$ IP=$(oc get -n istio-system service istio-ingressgateway -ojsonpath='{.status.loadBalancer.ingress[0].hostname}')
```

Comment:
Are we guaranteed to have a hostname set here for every OCP4 environment? Usually scripts branch between either ip or hostname (whatever is set).

Reply:
I don't know, actually. On the only supported platform (AWS), I expect so. But I agree we can probably do this bettererer

Reply:
We should do better! We should get real DNS here and not a host header.

Reply:
Easy enough on a shared cloud. I hate to ignore the local cluster peeps, though.

Reply:
Well, this issue is about OCP4, so we can assume AWS somewhere for now perhaps?

Reply:
An OpenShift Route with `dumpy.myproject.<my ocp cluster domain>` and an edit to Knative's config-network to change the domain from `example.com` to `<my ocp cluster domain>`?

Reply:
I think this might be fine getting in as is. We need to find the sequence of commands to make things externally available on OCP4 anyway, we can probably augment that in a subsequent PR if we want to.

Reply:
:+1: